subsys/bluetooth/mesh/Kconfig

```diff
@@ -812,7 +812,7 @@ config BT_MESH_LPN_INIT_POLL_TIMEOUT
 config BT_MESH_LPN_SCAN_LATENCY
 	int "Latency for enabling scanning"
 	range 0 50
-	default 15
+	default 4
```

Comment:
Probably this should be 2. Could you try to test it with 2?

Reply:
It works. I'll set 2.